src/ui/navigation/AppBars.tsx

```diff
@@ -55,18 +60,20 @@ export function MainAppBar({navigation, route, options}: NativeStackHeaderProps)
   }, [navigation]);
 
   return (
-    <AppHeader style={{backgroundColor: colors.primary}}>
-      {showMenu ? <AppBar.Action onPress={onActionLeftPress} icon={'menu'} color="white" /> : null}
-      <AppBar.Content
-        style={styles.contentContainer}
-        title={
-          <View style={styles.titleContainer}>
-            <Title style={styles.title}>Litentry</Title>
-          </View>
-        }
-      />
-      {options.headerRight ? options.headerRight({canGoBack: navigation.canGoBack()}) : null}
-    </AppHeader>
+    <>
+      <AppHeader style={{backgroundColor: colors.primary}}>
+        {showMenu ? <AppBar.Action onPress={onActionLeftPress} icon={'menu'} color="white" /> : null}
+        <AppBar.Content
+          style={styles.contentContainer}
+          title={
+            <View style={styles.titleContainer}>
+              <Title style={styles.title}>Litentry</Title>
+            </View>
+          }
+        />
+        {options.headerRight ? options.headerRight({canGoBack: navigation.canGoBack()}) : null}
+      </AppHeader>
+    </>
```

Comment:
Nothing seems to be added here, other than the empty fragment tags. Did you mean to add the `<NetworkConnectionError />` component here?

Reply:
Noo, It should not be added here.

Reply:
Why?